frontend/src/context/ApiCallsContext.jsx

```diff
@@ -64,10 +64,30 @@ export function ApiCallsContextProvider({ children }) {
       )
       .then((response) => response.data)
       .then((data) => {
-        const [first, second, third] = data.sort(
-          (a, b) => b.userscore - a.userscore
-        );
-        setScores([first, second, third]);
+        const [
+          first,
+          second,
+          third,
+          fourth,
+          fifth,
+          sixth,
+          seventh,
+          eighth,
+          ninth,
+          tenth,
+        ] = data.sort((a, b) => b.userscore - a.userscore);
+        setScores([
+          first,
+          second,
+          third,
+          fourth,
+          fifth,
+          sixth,
+          seventh,
+          eighth,
+          ninth,
+          tenth,
+        ]);
```

Comment:
if you're going for a top ten, take a look at slice: https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/slice